File: CodingSamples/Java/Runtime/GenClassTest/Program1.java
class Program {

	public static void main(String[] args) {
		SimpleStack<String> a = new SimpleStack<String>();
		a.push("Monday");
		a.push("Tuesday");
		a.push("Wednesday");
		a.push("Thursday");
		a.push("Friday");
		//a.push(23.4);
		while(!a.empty()){
			System.out.println(a.pop());
		}
		System.out.println("----------------------");
		SimpleStack<Interval> b = new SimpleStack<>(); 
		b.push(new Interval(4, 31));
		b.push(new Interval(6, 12));
		b.push(new Interval(5, 43));
		b.push(new Interval(3, 24));
		while(!b.empty()){
			System.out.println(b.pop());
		}
	}
}

File: CodingSamples/DotNet/Runtime/GenTypeTest/DemoApp/Program1.cs
class Program
{
    static void Main(string[] args)
    {
        SimpleStack<string> a = new SimpleStack<string>();
        a.Push("Monday");
        a.Push("Tuesday");
        a.Push("Wednesday");
        a.Push("Thursday");
        a.Push("Friday");
        //a.Push(9.8);
        while(!a.Empty())
            Console.WriteLine(a.Pop());
        Console.WriteLine("-------------------");
        SimpleStack<Interval> b = new SimpleStack<Interval>();
        b.Push(new Interval(5, 21));
        b.Push(new Interval(7, 42));
        b.Push(new Interval(6, 13));
        b.Push(new Interval(4, 54));
        while(!b.Empty())
            Console.WriteLine(b.Pop());
    }
}
